.circleci/config.yml

```diff
@@ -15,18 +15,18 @@ jobs:
 
     check-python-cs:
         docker:
-            - image: circleci/python:3.8
+            - image: cimg/python:3.7
```

Comment:
3.8 to 3.7?

Reply:
On these new images python3.8 is not available so I had to use 3.7.
3.10 is available though, maybe we want to use 3.10 instead ?